src/get-started/localisation/index.md.njk

```diff
@@ -0,0 +1,92 @@
+---
+layout: layout-pane.njk
+title: Using localisation with GOV.UK Frontend
+section: Get started
+theme: How to guides
+order: 5
+description: Localisation allows you to translate GOV.UK Frontend components into other languages
+---
+
+Localisation allows you to translate GOV.UK Frontend components into other languages
+
+## Localisation using Nunjucks
+
+If you’re using the Nunjucks macros provided by GOV.UK Frontend, you can localise the UI text that appears within components by passing translation strings as named parameters.
+
+We have provided specific localisation guidance for some of the components. These include:
+
+* [accordion](/components/accordion/)
+* [character count](/components/character-count/)
+* [error message](/components/error-message/)
+* [footer](/components/footer/)
+* [header](/components/header/)
+* [summary list](/components/summary-list/)
+
+## Localisation using Javascript
+
+You can only pass translation strings to JavaScript with a list of key-value pairs.
+
+
+You can also pass a `locale` property containing a [two-letter ISO 639-1 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes), which determines which plural forms and number formatting rules you use. If you do not include this, the `locale` defaults to the document language specified on the `html` element.
+
+### Passing translation strings through `initAll`
+
+You can pass translation strings for multiple components when initialising GOV.UK Frontend. All Frontend components will automatically use these translations, unless initialised separately.
+
+
+```javascript
+window.GOVUKFrontend.initAll({
+  accordion: {
+    i18n: {
+      locale: "cy",
+      translations: {
+        show_all_sections: "Dangoswch bob adran",
+        hide_all_sections: "Cuddiwch bob adran",
+        show_this_section: "Dangoswch yr adran hon",
+        hide_this_section: "Cuddio'r adran hon"
+      }
+    }
+  },
+  character_count: {
```

Comment:
Now that we've decided on two approaches to passing translations, with data-attributes being the primary approach, we could have an example of translating a component using data-attributes and then what that would look like using the JS approach.